charts/operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: prometheus-engine
+description: Google Managed Service for Prometheus (GMP) Operator
+type: application
+version: "0.1.0"
```

Comment:
question: what's the difference between `version` and `appVersion`?

Reply:
`version` is the version of the chart. `appVersion` is the version of GMP.

Reply:
ah yea ok